modules/persistent-storage-csi-azure-disk-perf-plus-create-new-disk.adoc

```diff
@@ -0,0 +1,28 @@
+//
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-azure.adoc
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-disk-perf-plus-create-new-disk_{context}"]
+= Enabling performance plus by creating new disks
```

Comment:
I think we should reword this to:
`Enabling performance plus by snapshot or clone`

Reply:
Done.